releases/forecastle.yaml

```diff
@@ -38,7 +41,15 @@ releases:
           deployment:
             annotations: ""
           config:
-            namespaces: [{{- env "FORECASTLE_NAMESPACES" | default "kube-system,monitoring" }}]
+            # If only running one instance of Forecastle, it is best to leave the instance name blank
+            instanceName: {{ env "FORECASTLE_INSTANCE_NAME" }}
```

Comment:
should we use required env?

Reply:
No. Normal case would be to leave `FORECASTLE_INSTANCE_NAME` unset, which leaves the instance unnamed and therefore the default.